src/utils.ts

```diff
@@ -34,3 +36,28 @@ export function getFillOpacity(selected: boolean, highlight: boolean, hasSelecti
 
     return DefaultOpacity;
 }
+
+export function getLegendFillOpacity(
+    selected: boolean,
+    hasSelection: boolean,
+    isHCM: boolean): number {
+
+    if ((hasSelection && !selected) && isHCM) {
```

Comment:
HCM is not a clear name. Lets use naming which will make sense for any person reading this code for a first time. 
e.g. - `isHighContrastMode` or `isHighContrast` as it is in colorHelper

Reply:
Done.